src/components/post/newPost/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment, useState, useEffect } from 'react';
+
+// Components
+import NewPostDisplay from './NewPostDisplay';
+
+// MUI
+import withStyles from '@material-ui/core/styles/withStyles';
+import Button from '@material-ui/core/Button';
+import style from '../../../style/style';
+
+// Icons
+import AddIcon from '@material-ui/icons/Add';
+
+// Redux
+import { connect } from 'react-redux';
+import { newUserPost } from '../../../redux/actions/dataActions';
+
+const NewPost = ({ classes = {}, UI = {}, newUserPost }) => {
+  const [open, setOpen] = useState(false);
+  const [error, setError] = useState({});
```

Comment:
See previous comment, probably don't need this if we're just basing stuff on UI.error anyways

Reply:
as mentioned above, fixed.